dev/run-python-skinny-tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Executes a subset of mlflow tests that is supported with fewer dependencies than the core mlflow package.
+# Tests include most client interactions and compatibility points with the mlflow plugins around tracking, projects, models, deployments, and the cli.
+
+# The SQL alchemy store's dependencies are added for a base client/store that can be tested against.
+# A different example client/store with a minimal dependency footprint could also work for this purpose.
+
+set -x
+# Set err=1 if any commands exit with non-zero status as described in
+# https://stackoverflow.com/a/42219754
+err=0
+trap 'err=1' ERR
+export MLFLOW_SKINNY='true'
+
+python -m pip uninstall sqlalchemy alembic sqlparse -y
```

Comment:
sqlalchemy was recently added to small-requirements.txt.

I can create a skinny-requirements.txt and use that to avoid further issues but it seemed like overkill.

Any suggestions?